packages/logs/src/domain/configuration.ts

```diff
@@ -28,11 +39,35 @@ export function validateAndBuildLogsConfiguration(
     return
   }
 
+  const allowedConsoleApis = Object.keys(ConsoleApiName) as ConsoleApiName[]
+  let forwardConsoleLogs: StatusType[] = []
+  if (isExperimentalFeatureEnabled('forward-logs') && initConfiguration.forwardConsoleLogs !== undefined) {
+    if (
+      (initConfiguration.forwardConsoleLogs !== 'all' && !Array.isArray(initConfiguration.forwardConsoleLogs)) ||
+      (Array.isArray(initConfiguration.forwardConsoleLogs) &&
+        initConfiguration.forwardConsoleLogs.some((api) => !includes(allowedConsoleApis, api)))
+    ) {
+      display.error(
+        `Forward Console Logs should be "all" or an array with allowed values ${allowedConsoleApis
+          .map((api) => `"${api}"`)
+          .join(', ')}`
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** to avoid an extra traversal of the array‏
```suggestion
        `Forward Console Logs should be "all" or an array with allowed values "${allowedConsoleApis.join('", "')}"`
```